packages/testing/src/test/chai-retry-plugin.unit.ts

```diff
@@ -217,6 +217,30 @@ describe('chai-retry-plugin', () => {
             await expect(resultFunction).retry().to.not.change(myObj, 'comas');
             expect(getCallCount()).to.equal(2);
         });
+
+        it('.respondTo()', async () => {
+            class EchoService {
+                echo: (() => void) | undefined;
+            }
+
+            setTimeout(() => {
+                EchoService.prototype.echo = () => {
+                    return;
+                };
+            }, 900);
+
+            await expect(() => new EchoService(), 'should respond')
+                .retry({ timeout: 1000 })
+                .to.respondTo('echo');
+        });
+
+        it('.keys()', async () => {
+            const arrayWithKeys = [{}, {}, {}, { a: 1, b: 2 }, { a: 1, b: 2, c: 3 }, { a: 1, b: 2, c: 3, d: 4 }];
+            const { resultFunction, getCallCount } = withCallCount((count) => arrayWithKeys[count]);
```

Comment:
there are many tests in this suite with code like this: `if (count === 3) return {a: 1, b: 2, c: 3}`. Why don't do the same here? I mean this array stuff is just confusing... without any benefit in my opinion.

Reply:
@yurii-ve added it here just to also test empty obj and partially correct result, you think there is no sense to verify such cases?

Reply:
in my opinion with such cases test that keys assertion works correctly and they are simply redundant if purpose of the test is to verify that retry plugin works correctly with keys assertion

Reply:
@yurii-ve fixed